server.js

```diff
@@ -2,34 +2,122 @@ import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
-
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
-
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
-
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(
+    {
+      Hello: "Welcome to this top-music API!",
+      Routes: [
+        { "/": "Startpage / Api Info" },
+        { "/allsongs": "all tracks-data" },
+        { "/songs/id/:id": "singel track" },
+        { "/songs": " first 5 songs (5 per page)" },
+        { "/songs?page=2&pageSize=5": "page 2 (5 per page)" },
+      ]
+    });
+});
+const { Schema } = mongoose;
+
+const songSchema = new Schema({
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+})
+const Song = mongoose.model("Song", songSchema);
+
+app.get("/songs/id/:id", async (req, res) => {
```

Comment:
It took my a while to understand that it was the hard-coded id in the database that was used and not the one generated by MongoDB maybe this should be mentioned in the descriptions?